### ISSUE TITLE
Does not pick up /script CastSpellByName("Spell_Name")

### ISSUE THREAD

**Issue by lurexx** [Aviana/HealComm#8]
Just updated to LUF ver 2200

However, i noticed that the /script CastSpellByName("Spell_Name") is not picked up in the visual "healing prediction".

I updated from LUF ver 2020, that used the 1.4 revision of HealComm, where the script is picked up.

PS. Luna Unit Frames is glorious! Thank you a lot for developing and continuing to improve it. You are the best!

**Comment by Aviana** [comment]
Works fine on my end...
make sure you
- don't use this addon when you are already using luna, you only need one
- double check your macros

**Comment by lurexx** [comment]
Hello again.

I was mistaken by creating an issue on repository, I was referring to the Library HealComm, not the standalone addon, sorry for creating confusion.

The issue remains. I wiped my client, deleted the wtf, wdb and interface contents etc. 
While using only Luna Unit Frames, I could still reproduce the error.

If I use:
/script CastSpellByName("Flash Heal") OR
/Cast Flash Heal

It will not show any visual healing indication, however if i use a /lunamouseover or clickcasting it will show it.

I tried to replace the Healcomm library in the "libs" folder to an older release version (1.4, the one I used before) were it works.

I also tried the stand alone HealComm (without Luna enabled ofc) where I had the same issue

If it works on your end, and I have a fresh client, might it be a server specific thing, hmm. ( I play on kronos)

Sorry for creating any inconvenience.
